src/hotspot/share/opto/escape.cpp

```diff
@@ -233,6 +243,9 @@ bool ConnectionGraph::compute_escape() {
 
   if (non_escaped_allocs_worklist.length() == 0) {
     _collecting = false;
+#ifndef PRODUCT
+    escape_state_statistics(java_objects_worklist);
+#endif
```

Comment:
You can use `NOT_PRODUCT()` macro for one line which you have a lot in these changes.

Reply:
NIT: Use NOT_PRODUCT